src/celeritas/ext/detail/ImportProcessConverter.cc

```diff
@@ -323,7 +332,13 @@ void ImportProcessConverter::store_em_tables(const G4VEmProcess& process)
         process_.models.push_back(model_class_id);
 
         // Save element-selector data
-        process_.micro_xs.insert({model_class_id, this->add_micro_xs(model)});
+        if (model_class_id == ImportModelClass::bethe_heitler_lpm
+            || model_class_id == ImportModelClass::klein_nishina
+            || model_class_id == ImportModelClass::livermore_rayleigh)
```

Comment:
Can you add a comment about why only these three models need the EM tables? Are you sure none of  the other muon (or other potential classes) need exporting?

Reply:
Good point, there are a couple planned models (eCoulombScattering, muPairProduction) that build element selectors -- i'll add those too.

Reply:
Since we're not able to transport protons in celeritas, then if we want to add the coulomb scattering process, the production cutoff should be effectively infinite, right? (I.e., protons of any energy get deposited locally via dE/dx.) Is there a way to get Geant4 to do this?

Reply:
Deferring to #485